reference/tools.rst

```diff
@@ -1383,9 +1385,11 @@ Methods:
     Returns the revision number corresponding to the last changed item in the working folder
     (``use_wc_root=False``) or in the working copy root (``use_wc_root=True``).
 - **get_branch()**:
-    Tries to deduce the branch name from the
-    `standard SVN layout <http://svnbook.red-bean.com/en/1.7/svn.branchmerge.maint.html>`_. Will
-    raise if cannot resolve it.
+    Tries to deduce the branch name from the `standard SVN layout <http://svnbook.red-bean.com/en/1.7/svn.branchmerge.maint.html>`_ and
+    returns the current branch name. Will raise if cannot resolve it.
+- **get_tag()**:
+    Tries to deduce the tag name from the `standard SVN layout <http://svnbook.red-bean.com/en/1.7/svn.branchmerge.maint.html>`_ and
+    returns the current tag name. Otherwise it will return ``None``. Will raise if cannot resolve it.
```

Comment:
I would remove the `raise` (or add it to the other functions too, but better to remove it, I think)